src/test/scala/temple/DSL/semantics/SemanticAnalyserTest.scala

```diff
@@ -6,23 +6,11 @@ import temple.DSL.semantics.AttributeType._
 import temple.DSL.syntax
 import temple.DSL.syntax.Arg._
 import temple.DSL.syntax.{Args, DSLRootItem, Entry}
+import temple.DSL.syntax.Entry.{Attribute => AttributeEntry}
```

Comment:
Renamed to avoid this line wrapping onto
```scala
Entry
  .Attribute(
    "a",
    // ...
```
https://github.com/TempleEight/temple/pull/81/files#diff-1a0467d6378d04f12e4aa1a1be41543fR110

Reply:
This feels like a lot of unnecessary faff for a bit of line wrapping